app/book-a-vaccination/2025/11/improving-contact-details/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Improving the contact details screen"
+date: 2025-11-03
+---
+
+The feedback from an accessibility audit on the National Booking Service (NBS) told us the way we collect contact details on NBS was not very accessible.
+
+###  The existing design
+
+The existing screen used four inputs to ask users to enter either a mobile number or email address, with double inputs required for both types of contact information.  
+
+Because we need to be able to give users the option of giving us a mobile number or an email address (or both, if they choose). All fields on the screen were marked as optional.   
```

Comment:
```suggestion
Because we need to be able to give users the option of giving us a mobile number or an email address (or both, if they choose). All fields on the screen were marked as optional.
```